west.yml

```diff
@@ -101,6 +101,10 @@ manifest:
       repo-path: sdk-mcuboot
       revision: v1.7.99-ncs4-rc2
       path: bootloader/mcuboot
+    - name: mbedtls-nrf
+      path: mbedtls
+      repo-path: sdk-mbedtls
+      revision: mbedtls-2.26.0 # Soon to be 3.0. Comment given to re-trigger CI.
```

Comment:
Please remove the comment

Reply:
Removed

Reply:
`git commit --amend --no-edit` is a good way to re-trigger CI. Just updates the timestamp I think.